examples/train_model.py

```diff
@@ -33,7 +33,7 @@
     project="example-script",
```

Comment:
I would revert changes to this file, so as to not change it as part of this PR unless there are functional changes that you think should be included.

Reply:
File has been reverted